spec/spec_helper.rb

```diff
@@ -13,6 +15,16 @@
 rescue LoadError
 end
 
+def yaml_loader(path, spec_name=nil)
+  file =  File.expand_path(path)
+  raise "File #{file} cannot be found!" unless File.exist?(file)
+  raw_config = ERB.new(File.read(file)).result
+  raise "No configuration text found!" if raw_config.empty?
+  data = YAML.load(raw_config)
+  raise "No configuration data found!" if data.empty?
+  spec_name ? data[spec_name] : data
+end
```

Comment:
I'm not sure how I feel about it, but what do two think about something like
```ruby
def yaml_loader(path, spec_name=nil)
  file =  File.expand_path(path).presence               or raise "File #{file} cannot be found!"
  raw_config = ERB.new(File.read(file)).result.presence or raise "No configuration text found!"
  data = YAML.load(raw_config).presence                 or raise "No configuration data found!"
  spec_name ? data[spec_name] : data
end
```
? Alternatively, I don't mind the one liner that was there. I think it's easy enough to put a binding and figure out what's going on, but I think we can at least get rid of the first error. `File.read` already errors in an obvious way
```ruby
[5] pry(main)> File.read('/foo')
Errno::ENOENT: No such file or directory @ rb_sysopen - /foo
from (pry):5:in `read'
```
Regardless, maybe this would be better off in a different PR?